CPS-xxxx/README.md

```diff
@@ -0,0 +1,108 @@
+```yaml
+---
+CPS: ????
+Title: Missing Merkle Root of Transactions in Cardano Block Header
+Status: Open
+Category: Ledger
+Authors:
+  - Alexander Slesarenko <alex.slesarenko@iohk.io>
+Proposed Solutions: []
+Discussions:
+Created: 2025-01-17
+License: CC-BY-4.0
+---
+```
```

Comment:
```suggestion
---
CPS: ????
Title: Missing Merkle Root of Transactions in Cardano Block Header
Status: Open
Category: Ledger
Authors:
  - Alexander Slesarenko <alex.slesarenko@iohk.io>
Proposed Solutions: []
Discussions:
Created: 2025-01-17
License: CC-BY-4.0
---
```
This is a text block with YAML syntax, not a YAML code block.  The code tags need to be removed for it to be parsed correctly: both by GitHub and downstream filters / renderers.